config/pickit.ini

```diff
@@ -35,12 +35,12 @@
 ;==================================================================;
 
 ;potions
-misc_rejuvenation_potion=0
+misc_rejuvenation_potion=1
```

Comment:
Don't change param defaults without really meaning to. Revert this file to original defaults and only add new fields